upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.8.yaml.template

```diff
@@ -0,0 +1,379 @@
+# This ConfigMap is used to configure a self-hosted Calico installation.
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: calico-config
+  namespace: kube-system
+data:
+  # The calico-etcd PetSet service IP:port
+  etcd_endpoints: "{{ $cluster := index .EtcdClusters 0 -}}
```

Comment:
If we do this we should probably handle TLS too since that config can provide it now.

Reply:
can you expand on this :)

Reply:
The new etcd configuration supports TLS, if someone happens to enable it there, we'll blow up the cluster unless we support it here.